client/scss/layouts/_compare-revisions.scss

```diff
@@ -1,9 +1,21 @@
-$color-addition-dark: #a6f3a6;
-$color-addition-light: #ebffeb;
-$color-deletion-dark: #f8cbcb;
-$color-deletion-light: #ffebeb;
+$color-addition-dark: var(--color-addition-dark);
+$color-addition-light: var(--color-addition-light);
+$color-deletion-dark: var(--color-deletion-dark);
+$color-deletion-light: var(--color-deletion-light);
 
 .comparison {
+  --color-addition-dark: #a6f3a6;
+  --color-addition-light: #ebffeb;
+  --color-deletion-dark: #f8cbcb;
+  --color-deletion-light: #ffebeb;
```

Comment:
Sorry to be nitpicky but could we call these something other than light/dark? It may cause confusion with the dark theme. 

Maybe addition-foreground/background addition-primary/secondary addition-highlight/contrast

?

Reply:
I think this pattern needs more revisiting than just renaming those variables, so chose to keep the existing variable names as-is. At least for me I don’t even understand why there’s two variables per type of diff to start with.

Reply:
For ref, the two-shade split was from way back when this was initially set up: https://github.com/wagtail/wagtail/blame/5770bc2a9598509c0182d9b82a50b1f4c10131d5/wagtail/wagtailadmin/templates/wagtailadmin/pages/revisions/compare.html. As far as I understand it was initially so diff types were communicated with a colored border as well as the background. Then the border as well as text decorations were removed later: f96321c1ce55d4b3128f51dba0dbc83255496423.